GSA-TTS/FAC
# 4023 adjust intaketodissemination logic for validation waivers

## Description
The FAC application is being updated to handle waivers that allow audit reports with exceptional situations, such as the absence of certifying auditors/auditees, to be submitted. This PR adds the necessary logic to enable these audit reports to pass all validations and be properly disseminated, even without a certifying auditor or auditee.

Related [ADR](https://github.com/GSA-TTS/FAC/issues/3977).

The code change includes:
- A Django admin interface for creating waivers.
- Logic to add all necessary placeholder data for the missing certifying auditor/auditee based on the type of waivers created.
- A test class to ensure the logic works as expected.

## How to test
1. Launch the stack locally and start a new submission until you lock the report as ready for certification (if you want to test the certifying auditor waiver or both. If you only want to test the certifying auditee waiver, then complete the auditor certification process and stop at the auditee certifying step).
2. Next, go to the Django admin [interface ](http://localhost:8000/admin) and add a new record (see picture below). Make sure to select the correct waiver types from the list.
3. After adding the waiver record, return to your submission page and refresh. The step should appear complete.


![image](https://github.com/GSA-TTS/FAC/assets/26260777/c012f840-3a16-4ef7-82e8-31bcdd03feba)


## PR checklist: submitters


## PR checklist: reviewers


The larger the PR, the stricter we should be about these points.
